### PR TITLE
Add RFC newsletter for March 2023 for #1689.

### PR DESCRIPTION
**Pull request by aj-stein-nist**
# Committer Notes

Add RFC Newsletter Archive and March 2023 edition.

Index page:

image

RFC 1 for 202303 page:

image

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/OSCAL/blob/main/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers
](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Do all automated CI/CD checks pass?

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- ~Have you written new tests for your core changes, as applicable?~
- [x] Have you included examples of how to use your new feature(s)?
- [x] Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.

### CODE REVIEW

File: docs/content/contribute/rfc-newsletter/newsletter-202303-03.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+8 hours
+
+**Value to OSCAL:**
+
+The community has asked for more tutorials for core OSCAL functionality.
+
+**I'm interested, how do I volunteer?**
+
+Visit the discussion board and comment here](https://github.com/usnistgov/OSCAL/discussions/1696).
```
Comment (nikitawootten-nist) [inline docs/content/contribute/rfc-newsletter/newsletter-202303-03.md:75]: B: Malformed markdown link

```suggestion
Visit the discussion board and comment [here](https://github.com/usnistgov/OSCAL/discussions/1696).
```

File: docs/content/contribute/rfc-newsletter/newsletter-202303-03.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+*This opportunity is subject to the outcome of the current effort, and may be delayed if additional detail is required to begin the draft.*
+
+**Value to OSCAL:** 
+
+This will help us enhance OSCAL models to communicate shared responsibility for controls without exposing the content of the entire SSP model.
+
+**I'm interested, how do I volunteer?**
+
+Visit the discussion board and comment here](https://github.com/usnistgov/OSCAL/discussions/1695).
```
Comment (nikitawootten-nist) [inline docs/content/contribute/rfc-newsletter/newsletter-202303-03.md:48]: B: Malformed markdown link

```suggestion
Visit the discussion board and comment [here](https://github.com/usnistgov/OSCAL/discussions/1695).
```

**Review by nikitawootten-nist — CHANGES_REQUESTED** [review]
See fixes below, otherwise looks good!

File: docs/content/contribute/rfc-newsletter/newsletter-202303-03.md (hunk 1 above)
Comment (aj-stein-nist) [reply docs/content/contribute/rfc-newsletter/newsletter-202303-03.md:75]: This is a good but oldie, I have been told the old ways of "click [here](#link)" is bad UX wise, but I should go for what I know. lol

**Review by nikitawootten-nist — APPROVED** [review]

**Review by iMichaela — APPROVED** [review]
Great work. Thank you, @aj-stein-nist .

**Pull request merged**